k8s/overlays/azure/kustomization.yaml

```diff
@@ -21,3 +21,10 @@ configMapGenerator:
 patchesStrategicMerge:
 - limit-gridexplore-cases.yaml
 - fluentd-containerd.yaml # default fluentd parse is for docker, on azure we use containerd
+- |-
+  apiVersion: v1
+  kind: Deployment
+  metadata:
+    name: gridadmin-app
+  $patch: delete
+  # We disable/don't deploy GridAdmin in the Azure/demo environment
```

Comment:
`Error: accumulating resources: accumulation err='accumulating resources from '../../overlays/azure': '/home/bouhoursant/Devin/gridsuite-deployment/bases/rte/mirror-deployment/k8s/overlays/azure' must resolve to a file': recursed accumulation of path '/home/bouhoursant/Devin/gridsuite-deployment/bases/rte/mirror-deployment/k8s/overlays/azure': no matches for Id Deployment.v1.[noGrp]/gridadmin-app.[noNs]; failed to find unique target for patch Deployment.v1.[noGrp]/gridadmin-app.[noNs]`
on ` kustomize build .`
Have you tested this?

Reply:
oups, there was a bad `apiVersion: v1` instead of  `app/v1`